app/assets/stylesheets/_user_research_recruitment_banner.scss

```diff
@@ -16,3 +16,10 @@
 .user-research-recruitment-banner__intro {
   color: govuk-colour("white");
 }
+
+.user-research-recruitment-banner__hide-this {
```

Comment:
I'm not sure what to think about this! Like, is there a reason we can't find anything in the design system to help us out here?